src/dfcx_scrapi/core/agents.py

```diff
@@ -339,7 +339,12 @@ def get_validation_result(
         return val_results_dict
 
 
-    def export_agent(self, agent_id: str, gcs_bucket_uri: str) -> str:
+    def export_agent(
+        self,
+        agent_id: str,
+        gcs_bucket_uri: str,
+        environment: str = None
```

Comment:
If we're planning to set `environment` = `None`, let's drop the assignment of `request.environment` behind an `if statement`.